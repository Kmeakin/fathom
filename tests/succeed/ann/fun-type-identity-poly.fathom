fun (A : Type) -> A -> A : Type
