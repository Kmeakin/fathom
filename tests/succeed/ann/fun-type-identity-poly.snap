stdout = '''
fun (A : Type) -> fun (_ : A) -> A : Type
'''
stderr = ''
